src/rexsdesktop/CurrentUser.java

```diff
@@ -18,26 +18,138 @@
  */
 public class CurrentUser {

```

Comment:
don't use static properties/functions for model properties. Instead use instance properties and functions. When you define models most of the time yo won't need static functions/properties

Reply:
In this case I need to access the current user data from almost everywhere in the program, I tried thinking of a way to achieve this without having static properties, but I didn't find an alternative solution, so what do you recommend me in this case?

Reply:
Use a [singleton](https://refactoring.guru/design-patterns/singleton) instead